src/services/api/admin/donate/correspondent-banks/correspondent-banks-api.ts

```diff
@@ -0,0 +1,32 @@
+import { AxiosInstance } from 'axios';
+import { API_ROUTES } from '../../../../../const/common/api-routes/main-api';
+import { CorrespondentBankDetailsType } from '../../../../../types/admin/donate';
+
+export const CorrespondentBankDetailsApi = {
+    create: async (
+        client: AxiosInstance,
+        bankDetails: Omit<CorrespondentBankDetailsType, 'id'>,
+    ): Promise<CorrespondentBankDetailsType> => {
+        const response = await client.post<CorrespondentBankDetailsType>(
+            API_ROUTES.DONATE.CORRESPONDENT_BANK_DETAILS,
+            bankDetails,
+        );
+        return response.data;
+    },
+
+    update: async (
+        client: AxiosInstance,
+        id: number,
+        bankDetails: Partial<CorrespondentBankDetailsType>,
+    ): Promise<CorrespondentBankDetailsType> => {
+        const response = await client.put<CorrespondentBankDetailsType>(
```

Comment:
Same question here, should it be `.put` or `.patch`?
```suggestion
        const response = await client.patch<CorrespondentBankDetailsType>(
```

Reply:
I agree with you, but on the backend we only have an endpoint with PUT.